src/web/components/Badge.tsx

```diff
@@ -14,41 +14,25 @@ const badgeWrapper = css`
     ${badgeSizingStyles}
 `;
 
-const svgStyles = css`
-    svg {
-        display: block;
-        width: auto;
-        ${badgeSizingStyles}
-    }
+const imageStyles = css`
+    display: block;
+    width: auto;
+    ${badgeSizingStyles}
 `;
 
 const badgeLink = css`
     text-decoration: none;
 `;
 
 type Props = {
-    svgSrc: () => JSX.Element;
-    linkTo?: string;
+    imageUrl: string;
+    seriesTag: string;
```

Comment:
I hope not, but does `seriesTag` need to be optional?

Reply:
na better not be. a badge *has* to have a series or it will never show